packages/react-native/lib/ecosystem-tools/ecosystemHooks.ts

```diff
@@ -41,7 +42,7 @@ export function useEcosystems({issuer, verifier, schemaId}) {
     if(isLoading) return;
     setIsLoading(true);
     setIsError(false);
-    getEcosystems({issuerDID: issuer, verifierDID: verifier, schemaId: schemaId})
+    getEcosystems({issuerDID: issuer, verifierDID: verifier, schemaId: schemaId, networkId: getWallet().getNetworkId()})
```

Comment:
is there a test we could add for this?

Reply:
We already have few tests for ecosystem tools https://github.com/docknetwork/react-native-sdk/blob/b41fafbd27908e63c1f636cf609227fb58b27246/integration-tests/ecosystem-tools.test.ts

Reply:
but none of them caught this error scenario?

Reply:
I can try to add some unit tests for those react hooks, but will be basically mocking the network calls.

Reply:
I just added a new unit test suite for the ecosystem hooks, it would catch this specific error